packaging/docker/run.sh

```diff
@@ -80,6 +80,7 @@ fi
 exec grafana-server                                         \
   --homepath="$GF_PATHS_HOME"                               \
   --config="$GF_PATHS_CONFIG"                               \
+  --packaging docker                                        \
```

Comment:
should there be an equal sign here?

Reply:
@torkelo bit works without too, but I'll add it in to make the changes more consistent.